### DISCUSSION TITLE
Can Download

### DISCUSSION THREAD

**Discussion by ghost** [cinemagoer/cinemagoer#353] [Ideas]
Can we download movie poster and trailer video using this module?

**Comment by alberanid** [comment]
Posters yes, video trailers, no.

For posters, you will get the "cover url" (if available) that is the small poster shown in the main page of a movie.  It can be converted to a full-size poster using a function in the imdb.helpers module.

Example:

```
#!/usr/bin/env python3

from imdb import IMDb, helpers

ia = IMDb()
utopia = ia.get_movie('2384811')
cover = utopia.get('cover url')
print('small cover:', cover)
print('full cover:', helpers.fullSizeCoverURL(utopia))
```
